src/core/dir.c

```diff
@@ -98,30 +104,42 @@ static void move_left(char *str)
     *str = 0;
 }
 
-const char *dir_get_case_corrected_file(const char *filepath)
+const char *get_case_corrected_file(const char *dir, const char *filepath)
```

Comment:
```suggestion
static const char *get_case_corrected_file(const char *dir, const char *filepath)
```

`static` is missing.